fnal_column_analysis_tools/lookup_tools/jme_standard_function.py

```diff
@@ -5,29 +5,60 @@
 from copy import deepcopy
 import numba
 
-from numpy import sqrt,log,exp
+from numpy import sqrt,log,exp,abs
 from numpy import maximum as max
 from numpy import minimum as min
+from numpy import power as pow
 
 def numbaize(fstr, varlist):
     """
         Convert function string to numba function
         Supports only simple math for now
         """
+    val = fstr
+    try:
+        val = float(fstr)
+        fstr = 'np.full_like(%s,%f)' % ( varlist[0], val )
+    except ValueError:
+        val = fstr
     lstr = "lambda %s: %s" % (",".join(varlist), fstr)
     func = eval(lstr)
-    nfunc = numba.njit(func)
-    return nfunc
+    #nfunc = numba.njit(func)
```

Comment:
I think the meaning of this function has drifted quite a bit

Reply:
quite a lot yes.